### PR TITLE
Unified mode

### PR DESCRIPTION
**Pull request by damacus**
### Description

Enables unified mode

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: resources/acl.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@
 property :config_dir, String, default: '/etc/haproxy'
 property :config_file, String, default: lazy { ::File.join(config_dir, 'haproxy.cfg') }
 
+unified_mode true
```
Comment (JohnRoesler) [inline resources/acl.rb:7]: does this break chef 14 builds?

Comment (tas50) [reply resources/acl.rb:7]: Everything but the very latest chef 14

Comment (damacus) [reply resources/acl.rb:7]: Ah, funnily enough I only tested it on the latest 14

**Comment by damacus** [comment]
This needs a metadata.rb change to 15

**Pull request merged**